mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -599,17 +599,12 @@ def test_transpiled_with_export_to_xlsx_format_two_sheets():
     mito.set_dataframe_format(0, DF_FORMAT_HEADER)
     mito.set_dataframe_format(1, DF_FORMAT_HEADER_AND_ROWS)
     filename = 'test_format_two.xlsx'
-    mito.export_to_file('excel', [0, 1], filename)
+    mito.export_to_file('excel', [1], filename)
     assert "\n".join(mito.transpiled_code) == """from mitosheet.public.v3 import *
 import pandas as pd
 
 with pd.ExcelWriter(r\'test_format_two.xlsx\', engine="openpyxl") as writer:
```

Comment:
Test looks good! This is one of the issues with copying generated code back into a test -- it can be easy to not read thoroughly and then miss an important bit.